src/gui/tray/UserModel.cpp

```diff
@@ -829,12 +822,25 @@ QImage ImageProvider::requestImage(const QString &id, QSize *size, const QSize &
     Q_UNUSED(size)
     Q_UNUSED(requestedSize)
 
-    if (id == "currentUser") {
-        return UserModel::instance()->currentUserAvatar();
-    } else {
-        int uid = id.toInt();
-        return UserModel::instance()->avatarById(uid);
+    const auto makeIcon = [](const QString &path) -> QImage {
```

Comment:
Funny that you had to specify the return type here, I'd have expected this to be deduced by the compiler.

Reply:
Turns out it can be deduced after all